test/e2e/tests/commands/clean.test.ts

```diff
@@ -20,22 +19,11 @@ suite("task - clean", function () {
 
     // Run clean task
     await vscode.commands.executeCommand("workbench.action.tasks.runTask", {
-      task: "hardhat: clean main",
+      type: "hardhat",
+      task: "clean",
```

Comment:
Q: The new task is no longer just `main`. Is this intended?

Reply:
Yes, because it had the additional "projects/main" label rendered in the item (I presume it was a nice thing to do for the fuzzy searcher?), which we don't get anymore in this version and that's the only version it's used with.